### PR TITLE
Merge-worthy 2.74

### PR DESCRIPTION
**Pull request by wanda-carlson**
Merge-worthy 2.74

### CODE REVIEW

**Comment by wanda-carlson** [comment]
This is ok!
Screen Shot 2022-01-06 at 2 57 26 PM

**Comment by wanda-carlson** [comment]
Hi **Slack user**

**Comment by henry-richard** [comment]
Leave it here.

File: README.md [hunk 1]
```diff
@@ -29,3 +29,4 @@ Merge-worthy 2.6
 Merge-worthy 2.71
 Merge-worthy 2.72
 Merge-worthy 2.73
```
Comment (henry-richard) [inline README.md:31]: what is wrong with the versions?

File: README.md [hunk 2]
```diff
@@ -29,3 +29,4 @@ Merge-worthy 2.6
...
+Merge-worthy 2.74
```
Comment (henry-richard) [inline README.md:32]: I think its fine
Screen Shot 2022-01-10 at 9 44 25 AM

**Review by henry-richard — CHANGES_REQUESTED** [review]
make some changes, add a new file

**Review by henry-richard — APPROVED** [review]

**Pull request merged**